charts/gitops-server/templates/test-user.yaml

```diff
@@ -41,9 +41,9 @@ rules:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: cluster-user-auth
-  namespace: {{.Values.namespace}}
+  name: wego-test-user-auth
 type: Opaque
 data:
-  username: {{.Values.adminUsername}}
-  password: {{.Values.adminPassword}}
+  username: {{ .Values.testUser.username | b64enc | quote }}
+  password: {{ .Values.testUser.password | b64enc | quote }}
```

Comment:
do these need to be `b64enc` here? i think i remember that k8s will do that automagically when the secret is created

Reply:
you do (I just checked against a kind cluster with a simple helm chart). I think the auto-b64encode is only when you use `kubectl`

Reply:
i swear i have seen that as well when just doing like a:

```go
secret := &v1.Secret{
        ObjectMeta: metav1.ObjectMeta{
                Name:      "foo",
                Namespace: "bar",
        },
        Type: "Opaque",
        Data: map[string][]byte{
                "key": []byte("value"),
        },
}

return yaml.Marshal(secret)
```
even then it is encoded

Reply:
bold of you to assume there're rules ;)